tests/auth_tests/test_basic.py

```diff
@@ -73,6 +73,22 @@ def test_superuser(self):
         self.assertTrue(super.is_active)
         self.assertTrue(super.is_staff)
 
+    def test_superuser_no_email_or_password(self):
+        cases = [
+            {},
+            {'email': ''},
+            {'email': None},
+            {'password': None},
+        ]
+        for i, kwargs in enumerate(cases):
+            with self.subTest(**kwargs):
+                super = User.objects.create_superuser(
```

Comment:
I would try to avoid `super`, maybe `superuser`.

This can be single-lined:
```python
superuser = User.objects.create_superuser('super%s' % i, **kwargs)
```

Reply:
OK, renamed. (But `super` is used throughout this file...)

Reply:
IMO, we should try to avoid overriding builtins like `super` or `object` (I would accept such cleanup :wink: ).